apps/platform-app/src/app/[locale]/(profile)/(components)/Header/Header.tsx

```diff
@@ -146,27 +148,31 @@ async function Header({ buttonsVariant = 'main' }: HeaderProps) {
     }
   }
   return (
-    <header className={styles.wrapper}>
-      <Container>
-        <div className={styles.headerContent}>
-          <div className={styles.logoAndGhStarsWrapper}>
-            <Link href={AppRoutes.home} className={styles.logo}>
-              <img src={logo.src} alt="Logo" />
-              <h1 className={styles.title}>Good Dev Hunting</h1>
-            </Link>
-            <GitHubStarsButton />
+    <ThemeWrapper>
```

Comment:
Using the theme wrapper in the page/layout will eliminate the need for it in this component